### PR TITLE
add DSC logo img

### LINKED ISSUE #4: logo favicon setting
logo favicon setting

### PR DESCRIPTION
**Pull request by MOBUMIN**
add DSC logo img

Resolve #4

### CODE REVIEW

**Comment by Enoch-Kim** [comment]
good job!

**Review by Enoch-Kim — APPROVED** [review]
good job

**Comment by Enoch-Kim** [comment]
> good job

**Review by Enoch-Kim — CHANGES_REQUESTED** [review]
cancel approve

File: _layouts/default.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <link rel="icon" href="{{ site.baseurl }}/assets/images/logo.png">
+    <link rel="icon" href="{{ site.baseurl }}/assets/images/dsclogo.jpeg">
```
Comment (jeongukjae) [inline _layouts/default.html]:
```suggestion
    <link rel="icon" href="{{ site.baseurl }}/{{ site.logo }}">
```

**Review by jeongukjae — CHANGES_REQUESTED** [review]

**Review by jeongukjae — DISMISSED** [review]
LGTM :)

File: _config.yml [hunk 2]
```diff
@@ -6,8 +6,8 @@ baseurl:
 include: ["_pages"]
 permalink: /:title/
 
-logo: 'assets/images/logo.png'
-favicon: 'assets/images/logo.png'
+logo: 'assets/images/dsclogo.png'
+favicon: 'assets/images/dsclogo.png'
```
Comment (Enoch-Kim) [inline _config.yml]: * remove favicon
```suggestion
```

**Review by Enoch-Kim — APPROVED** [review]
LGTM

**Review by jeongukjae — APPROVED** [review]

**Pull request merged**